minicluster/src/main/java/tachyon/master/AbstractLocalTachyonCluster.java

```diff
@@ -57,12 +61,21 @@
 
   protected BlockWorker mWorker;
   protected LineageWorker mLineageWorker;
+  protected UnderFileSystemCluster mUfsCluster;
 
   protected String mTachyonHome;
   protected String mHostname;
 
   protected Thread mWorkerThread;
 
+  /** The names of all the master services, for creating journal folders. */
+  protected String[] mMasterServiceNames = new String[] {
```

Comment:
maybe as a TODO, can we consolidate this with the similar array defined in `Format.java`?